packages/integration-tests/src/EntityManager.factories.test.ts

```diff
@@ -86,6 +86,26 @@ describe("EntityManager.factories", () => {
     expect(b1.author.get.firstName).toEqual("a3");
   });
 
+  it("can create a author and set mentor automatically", async () => {
+    const em = newEntityManager();
+    // Given there is an existing author
+    const a1 = newAuthor(em);
+    // When we create the second author
+    const a2 = newAuthor(em);
+    // Then we expect it to be mentored by the first
+    expect(a2.mentor.get).toBe(a1);
+  });
+
+  it("can create a author and configure to not set the mentor", async () => {
+    const em = newEntityManager();
+    // Given there is an existing author
+    newAuthor(em);
+    // When we create the second author
+    const a2 = newAuthor(em, { associateOptionalEntities: { mentor: false }});
```

Comment:
```
    const a2 = newAuthor(em, { mentor: null });
```

Also works for disabling the auto-creation and I think specifies the same intent, i.e. "please keep the `mentor` not set".

Reply:
Ah yes, thanks you! That does work 👍